modules/cluster-logging-rn-5.6.4.adoc

```diff
@@ -0,0 +1,32 @@
+
+* Before this update, when the `tls.insecureSkipVerify` option was set to `true`, the Cluster Logging Operator would generate incorrect configuration. As a result, the operator would fail to send data to Elasticsearch when attempting to skip certificate validation. With this update, the Cluster Logging Operator generates the correct TLS configuration even when `tls.insecureSkipVerify` is enabled. As a result, data can be sent successfully to Elasticsearch even when attempting to skip certificate validation. (link:https://issues.redhat.com/browse/LOG-3475[LOG-3475])
+
+* Before this update, when structured parsing was enabled and messages were forwarded to multiple destinations, they were not deep copied. This resulted in only some of the received logs including the structured message, while others did not. With this update, the configuration generation has been modified to deep copy messages before JSON parsing. As a result, all received messages now have structured messages included, even when they are forwarded to multiple destinations. (link:https://issues.redhat.com/browse/LOG-3640[LOG-3640])
+
+* Before this update, if the `collection` field contained `{}` it could result in the Operator crashing. With this update, the Operator will ignore this value, allowing the operator to continue running smoothly without interruption. (link:https://issues.redhat.com/browse/LOG-3733[LOG-3733])
+
+* Before this update, the `nodeSelector` attribute for the Gateway component of LokiStack did not have any effect. With this update, the `nodeSelector` attribute functions as expected. (link:https://issues.redhat.com/browse/LOG-3783[LOG-3783])
+
+* Before this update, the static LokiStack memberlist configuration relied solely on private IP networks. This resulted in LokiStack pods crashlooping when the cluster pod network was configured with a non-private IP network range. With this update, the LokiStack administrator can now choose to use the pod network for the memberlist configuration. This resolves the issue and prevents LokiStack pods from crashlooping when the cluster pod network is configured with a non-private IP network range. (link:https://issues.redhat.com/browse/LOG-3814[LOG-3814])
+
+* Before this update, if the `tls.insecureSkipVerify` field was set to `true`, the Cluster Logging Operator would generate an incorrect configuration. As a result, the Operator would fail to send data to Elasticsearch when attempting to skip certificate validation. With this update, the Cluster Logging Operator generates the correct TLS configuration even when `tls.insecureSkipVerify` is enabled. As a result, data can be sent successfully to Elasticsearch even when attempting to skip certificate validation. (link:https://issues.redhat.com/browse/LOG-3838[LOG-3838])
+
```

Comment:
I think we might need to add [LOG-3763](https://issues.redhat.com/browse/LOG-3763)

Reply:
Added